saveSystem.ts

```diff
@@ -0,0 +1,60 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+export interface JSONObject {
+  [key: string]: any;
+}
+
+class saveSystem {
```

Comment:
I think I'd like for us to have this a little more abstracted, let me give it a shot